views/guide.lavender

```diff
@@ -138,8 +138,8 @@ block content
             | becomes
             pre
               code.
-                &lt;section id="some_id" class="foo bar" attribute1="foo" 
-                attribute2="bar"&gt;&lt;/section&gt;
+                &lt;section id='some_id' class='foo bar' attribute1='foo' 
+                attribute2='bar'&gt;&lt;/section&gt;
```

Comment:
I had problems with this trying to execute itself, even when inside the code tag. I think this is a problem carrying over from the language itself?